lr_face/models.py

```diff
@@ -90,7 +90,14 @@ def predict_proba(self, X: List[FacePair]) -> np.ndarray:
         for pair in X:
             embedding1 = self.embedding_model.embed(pair.first, cache_dir)
             embedding2 = self.embedding_model.embed(pair.second, cache_dir)
-            score = np.linalg.norm(embedding1 - embedding2)
+            if embedding1 is None or embedding2 is None:
+                # If face was not detected, score = -1
+                score = -1
+                scores.append([score, 1 - score])
+            else:
+                score = spatial.distance.cosine(embedding1, embedding2)
+                scores.append([score, 1 - score])
+            # score = np.linalg.norm(embedding1 - embedding2)
```

Comment:
is there a reason you are now using spatial.distance.cosine in stead off np.linalg.norm?